juriscraper/pacer/email.py

```diff
@@ -45,3 +52,71 @@ def _get_date_filed(self):
         return convert_date_string(
             date_filed[0].lower().replace("filed on ", "")
         )
+
+    def _get_document_number(self):
+        document_number = self._get_document_number_from_link()
+        return (
+            document_number
+            if document_number is not None
+            else self._get_document_number_from_text()
+        )
```

Comment:
Can you unroll this so it reads top to bottom? I don't think we gain clarity from the complex return statement.